rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java

```diff
@@ -1117,7 +1123,8 @@ private int scheduleCancellation(Runnable runnable, Long timeLimit) {
         return Response.Status.OK.getStatusCode();
     }
 
-    private void abortLRA() {
+    // abortLRA is package private because it gets used by one of the byteman rules
+    void abortLRA() {
```

Comment:
I am not super keen on this part unless it is very much needed, couldn't we use reflect in the ByteMan helper to open up this method so the test can call it rather than promote the abortLRA into the package accessible API? I will try to look into this myself and see if I can avoid needing to drop the private and share something with you.

Reply:
I don't know how to do that.

Reply:
@mmusgrov I managed to work something out here: https://github.com/mmusgrov/narayana/pull/20 - please can I ask you to take a look?

Reply:
@tomjenkinson but I am open to some help on how to do the reflection.

Reply:
Great, thank you. Hopefully that code I provided in the pull request I mentioned (https://github.com/mmusgrov/narayana/pull/20) will be helpful here

Reply:
Ah right, doing it that, sure ok then.